src/auth-users/dto/sign-up/sign-up-email.dto.ts

```
@@ -0,0 +1,11 @@
+import {IntersectionType} from '@nestjs/mapped-types'
+import { SignUpDto } from './sign-up.dto';
+import { IsEmail, IsString } from 'class-validator';
+export class SignUpEmailDto extends IntersectionType(SignUpDto){
```

ESLint

[error] Insert `·`

(prettier/prettier)